webpack/stories/components/StoryWithCustomState/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Provider } from 'react-redux';
+import { action } from '@theforeman/stories';
+import Story from '../Story';
+
+// A super-simple mock of a redux store with a custom state.
+const StoryWithCustomState = ({ state, children }) => {
```

Comment:
nice! but I think it should move to `@foreman/stories` including some documentation